nemo/backends/pytorch/common/losses.py

```diff
@@ -95,10 +101,14 @@ def _ctc_loss(self, log_probs, targets, pad_mask):
         return loss
 
 
-class CrossEntropyLoss(LossNM):
+class CrossEntropyLossNM(LossNM):
     """
     CrossEntropyLoss
-
+    Args:
```

Comment:
these args should correspond to __init__'s args

Reply:
updated.